backend/src/main/java/gov/cdc/usds/simplereport/api/organization/OrganizationResolver.java

```diff
@@ -132,4 +133,25 @@ public FacilityStats facilityStats(@Argument UUID facilityId) {
   public List<UUID> getOrgAdminUserIds(@Argument UUID orgId) {
     return _organizationService.getOrgAdminUserIds(orgId);
   }
+
+  /**
+   * Used for outreach purposes - emails a CSV of org admin emails for the following type: 1.
+   * "facilities" - orgs that have facilities in the state 2. "patients" - orgs outside the state
+   * that have test results for patients whose address is in the state
+   *
+   * <p>The generated CSV is sent to the outreachMailingListRecipient email
+   *
+   * @param type "facilities" or "patients"
+   * @param state State abbreviation e.g. "NJ", "MN"
+   */
+  @MutationMapping
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public Integer sendOrgAdminEmailCSV(@Argument String type, @Argument String state) {
+    List<String> acceptableTypes = List.of("facilities", "patients");
+    if (acceptableTypes.contains(type.toLowerCase())) {
+      return _organizationService.sendOrgAdminEmailCSV(type, state);
+    } else {
```

Comment:
is there any way we could do something similar here for state param matching? Might make the process of interacting with this mutation easier if we somehow sanitize / check the format (and maybe the values to be within the set of active states if simple?) here like we do with type.

Reply:
ooo good idea! Will work on this as well! Thank you!